Editor/AGS.Types/EditorFeatures/MenuCommand.cs

```diff
@@ -49,6 +49,7 @@ public MenuCommand(string itemID, string itemName, Keys shortcutKey, string icon
 		private bool _checked;
 		private Keys _shortcutKey = Keys.None;
         private string _shortcutKeyDisplayString = string.Empty;
+		public List<MenuCommand> SubCommands { get; set; }

```

Comment:
Please, move this SubCommands out away from the list of "private" fields, to make it stand out as a public property. This is just to help notice it.

Reply:
I added a private variable with it and moved a public getter. Remade so it can only be set on creation.

Reply:
> Remade so it can only be set on creation.

I don't think that matters tbh, all other properties have public setters, and if you give out List<MenuCommand> then this collection may be modified anyway.